### DISCUSSION TITLE
Which fields does the search in mealie search?

### DISCUSSION THREAD

**Discussion by ovizii** [mealie-recipes/mealie#3643] [Q&A] [answered]
The reason I am asking is that the results look weird to me. 

i.e. searching for caramel yields 5 recipes which contain the word caramel in one of these fields:
- title
- description
- instruction step details
- ingredients

BUT a 6th recipe is not found where the word caramel is contained within the “ingredients section title” OR the “instructions section title” — this recipe also contains the word caramel inside instruction step details.

**Comment by ovizii** [comment]
One more weird thing to add:

I searched for "mayo” and found 4 recipes in my instance. 3 I was expecting to find, the 4th one is a cake recipe and has no occurrence of "mayo” anywhere in its content.

**Reply by RealFoxie** [reply, marked as answer]
The search might do some fuzzy searching (maybe only in postgress, not sure). So mayo might look similar to something else in that recipe. Or a word containing mayo might be included?

**Reply by ovizii** [reply]
Definitely no "mayo" in this recipe, its a cake recipe ;-) 
After you mentioned fuzzy search, I double-checked and it contains this text which may have triggered? 

**You may want to double your batch of ...**

**Reply by RealFoxie** [reply]
What version of mealie are you on and are you using postgress or mysql?
either way I think it is a valid comment they it should also search in section titles. I just don't think a lot of people have this problem, as often those titles have words that will exist somewhere else

**Reply by ovizii** [reply]
I'm using PostgreSQL and using mealie version v1.7.0

**Reply by RealFoxie** [reply]
Right, then the fuzzy matching is indeed the likely reason 👍
